packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html

```diff
@@ -0,0 +1,216 @@
+<div class="container-fluid px-0">
+    <div class="text-center" data-ng-if="!$ctrl.loading.init">
+        <oui-spinner data-size="l"></oui-spinner>
+    </div>
+
+    <div data-ovh-alert="cdnSharedSettings"></div>
+
+    <div data-ng-if="$ctrl.loading.init">
```

Comment:
Maybe the data that needs loading should be in the state resolve so we can benefit from the uniformized loading